### ISSUE TITLE
508-defect-2 [HEADINGS, COGNITION]: Headings should be logically ordered 

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22883]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _First name only. Delete if not applicable._

## Screenshots or Trace Logs

<img width="1383" alt="Screen Shot 2021-04-07 at 4 23 29 PM" src="https://user-images.githubusercontent.com/14154792/113929585-3d1fb280-97be-11eb-9f52-0388b889093a.png">
